bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/SatelModule.java

```diff
@@ -271,15 +271,15 @@ private SatelMessage readMessage() {
 
                 // if meanwhile thread has been interrupted, exit the loop
                 if (Thread.interrupted()) {
```

Comment:
Was there a reason to not change this line to `!Thread.currentThread().isInterrupted()` as below?